### PR TITLE
Update blacklist item property structure to add only relevant properties #2419

### PR DESCRIPTION
**Pull request by Shivan-997**
# Update blacklist item property structure to add only relevant properties

closes: #2419 

## Description

Fix Issue regarding duplicate blacklist objects by updating blacklistItem interface.

### CODE REVIEW

File: visualization/app/codeCharta/ui/nodeContextMenu/nodeContextMenu.component.spec.ts [hunk 1]
```diff
@@ -441,6 +441,15 @@ describe("nodeContextMenuController", () => {
 
 			expect(dialogService.showErrorDialog).toBeCalled()
 		})
+
+		it("should prevent duplicate blacklist object regarding issue #2419", () => {
```
Comment (ce-bo) [inline visualization/app/codeCharta/ui/nodeContextMenu/nodeContextMenu.component.spec.ts:445]: It is not necessary to refer to the issue number. The reference is placed in the commit message anyway. Furthermore, I don't want to review the issue to be able to understand the test case :-)

Comment (BridgeAR) [reply visualization/app/codeCharta/ui/nodeContextMenu/nodeContextMenu.component.spec.ts:445]: You are correct that the code references the issue. The test itself does not and if I want to know more about that, it's easier to identify the corresponding issue from a failing test case. :-)
Having self explanatory tests is always ideal. For regression tests, it's often still useful to know why a test case exists.

Comment (ce-bo) [reply visualization/app/codeCharta/ui/nodeContextMenu/nodeContextMenu.component.spec.ts:445]: The test case is well described, so I am fine with keeping the (helpful) reference, of course.
It's just a personal preference.

**Review by BridgeAR — APPROVED** [review]

**Comment by BridgeAR** [comment]
The e2e tests are failing.

**Pull request merged**